sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImpl.java

```diff
@@ -1138,18 +1136,7 @@ public void unpin() throws Exception {
         @Override
         public ApiFuture<AppendRowsResponse> appendRows(
             long offset, ProtoRows rows, Descriptor descriptor) throws Exception {
-          final AppendRowsRequest.ProtoData data =
-              AppendRowsRequest.ProtoData.newBuilder()
-                  .setWriterSchema(
-                      ProtoSchema.newBuilder().setProtoDescriptor(descriptor.toProto()).build())
-                  .setRows(rows)
-                  .build();
-          AppendRowsRequest.Builder appendRequestBuilder =
-              AppendRowsRequest.newBuilder().setProtoRows(data).setWriteStream(streamName);
-          if (offset >= 0) {
-            appendRequestBuilder = appendRequestBuilder.setOffset(Int64Value.of(offset));
-          }
-          return streamWriter.append(appendRequestBuilder.build());
+          return streamWriter.append(rows, offset);
```

Comment:
This change ignores `descriptor`. What's a good migration?

https://github.com/googleapis/java-bigquerystorage/pull/924/files

Reply:
@yayi-google (I saw your name on https://github.com/googleapis/java-bigquerystorage/pull/924/files). I appreciate if you can help here. Do you know a good way to adjust this Beam code for the method removal?
(I don't use Beam or Bigquery; I just want to upgrade the library in Beam)